### ISSUE TITLE
About test dataset

### ISSUE THREAD

**Issue by Alvacv** [JimmyChame/LSFNet#5]
How to choose 70 images from SID dataset to form the test set?

**Comment by JimmyChame** [comment]
Do you mean the training dataset? You can follow the code in Dataset to generate training data.

**Comment by JimmyChame** [comment]
I used the groundtruth data of val and test dataset in SID to synthesize the test set.
